drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/resources/org/drools/workbench/screens/guided/dtable/client/widget/analysis/Score Achievements.gdst

```diff
@@ -0,0 +1,167 @@
+<decision-table52>
+  <tableName>Score Achievements</tableName>
+  <rowNumberCol>
+    <hideColumn>false</hideColumn>
+    <width>-1</width>
+  </rowNumberCol>
+  <descriptionCol>
+    <hideColumn>false</hideColumn>
+    <width>-1</width>
+  </descriptionCol>
+  <metadataCols/>
+  <attributeCols/>
+  <conditionPatterns>
+    <Pattern52>
```

Comment:
Added one here https://github.com/droolsjbpm/drools-wb/pull/197/files#diff-b9aa6eac4f2a2c852697832bbdfe5cafR204

This Score Achievements.gdst is 1:1 with what Edson used.